webapp-django/crashstats/cron/management/commands/cron.py

```diff
@@ -0,0 +1,464 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import contextlib
+import datetime
+import logging
+import re
+import sys
+import time
+import traceback
+
+import markus
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from crashstats.cron.models import Job, Log
+
+
+logger = logging.getLogger('crashstats.crontabber')
+
+metrics = markus.get_metrics('crontabber')
+
+
+# NOTE(willkg): Times are in UTC. These are Django-based jobs. The other
+# crontabber handles the jobs that haven't been converted, yet.
+#
+# NOTE(willkg): You can't have the same job in here twice with two different
+# frequencies. If we ever need to do that, we'll need to stop using "cmd" as
+# the key in the db.
+3
+# cmd, frequency, time, backfill
+JOBS = [
+    {
+        # Test cron job
+        'cmd': 'crontest',
+        'frequency': '1d',
+    },
+]
```

Comment:
Oh, hrm. I messed up. This needs to support arbitrary arguments to pass to the command. For example, with `auditgroups`, we want to pass in `--persist`.